docs/UnityGroceriesRealWorld.md

```diff
@@ -0,0 +1,41 @@
+# UnityGroceries-RealWorld Dataset
+
+As part of this project, we collected a real world dataset containing 1267 images of the 63 target grocery items. This dataset is primarily used to assess the real-world performance of models trained on synthetic data with a model trained on real data. This dataset can be downloaded [here](https://storage.googleapis.com/datasetinsights/data/groceries/v3.zip).
+
+## Dataset Collection
+
+To take the photos for the dataset, we devised a strategy to vary object selection, placement, lighting, and background. We found that we, the humans, needed more humans and a better strategy to randomize more thoroughly and faster. This new strategy is what is outlined.
```

Comment:
> We found that we, the humans, needed more humans
lol! This was a joke actually. It is not a serious-sounding sentence :) how about "We iterated on our strategy a handful of times throughout the process to maximize our randomness and speed of data collection. The final strategy which we used for most of the images is described below."

Reply:
That's much better!